bfxapi/examples/ws/cancel_order.py

```diff
@@ -1,8 +1,9 @@
 import os
 import sys
+from tkinter import E
```

Comment:
Hello, for what is this used?

Reply:
Oh no! That looks like a runaway import! I will clean up both my PRs ASAP :)